Update apiary.apib

hakob9315 opened the pull request:
@hakob9315 Please name with corresponding story number. In this case, I suppose it should be RA-1209, right ?

hakob9315 committed:
Update apiary.apib

DimaUlyanets commented on apiary.apib:
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
-        {status:0 , msg:"Only organization administrators can update referrals."}           
+        {status:0 , msg:"Only organization administrators can update referrals."}        
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]        
```
@hakob9315  This url isn't correct, please be more attentive.

Joran182to commented on apiary.apib:
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]        
+
++ Response 200 (application/json)
```
@hakob9315 here we have `subscriber_date`, `created_at`, `updated_at`. I need a date of registration and date of subscription for 'confirmed' referrals and date of registration only for 'potential' referrals. Please describe what is what.
Also, this response should contain `user_name` and `is_new`.
`is_new` - boolean. It should let me know if user already seen this referral. So, if specific referral was not included in this response yet then user doesn't know about him and this referral should have `is_new`: `true`.

hakob9315 replied:
created_at  date of registration
subscriber_date  date of subscription

Joran182to commented on apiary.apib:
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
+
++ Response 200 (application/json)
+        {
+           "Potential": [
```
@hakob9315 let's keep it in solid style, so rename it please `Potential` -> `potential` . Same for `Confirmed` -> `confirmed`

hakob9315 replied:
ok

Commit:
Endpoints added

Please note, that this PR is based on https://github.com/DimaUlyanets/graspe-api-docs/pull/7 apiary file version. 

Please note, that `GET` `/organisation/{id_or_url}` and `POST` `/users/` endpoints are already exists, but changes to this endpoints are required.

Also, seems like 'Get referral system' endpoint returns only list of rewards. Please update information about all of endpoint you created in https://bitbucket.org/Dima_Ulyanets/graspe/pull-requests/297 . I believe that you already created endpoint to receive referral points number, new referral points number, referral code and user's rewards history so I didn't describe it in this PR. Please add it to apiary if so.

hakob9315 committed:
Update apiary.apib

hakob9315 committed:
Merge pull request #8 from DimaUlyanets/RA-1209-update

Endpoints added
